rest-api/dao/site_dao.py

```diff
@@ -19,6 +19,7 @@ class _FhirSite(FhirMixin, BackboneElement):
     FhirProperty('display_name', str, required=True),
     FhirProperty('mayolink_client_number', long),
     FhirProperty('site_status', str, required=True),
+    FhirProperty('enrolling_status', str, required=True),
```

Comment:
If you're going to make this required, we have to make sure to update the data in the database to have this column populated for all sites before we push these changes out... so like in the Alembic migration, set enrollingStatus to INACTIVE for all existing sites or something.

Alternatively, make this not required, and handle the None case below.

Reply:
done